ccf_transaction_processor/scripts/register_enclave_expected_measurements.py

```diff
@@ -25,16 +25,17 @@
 from ccf.clients import Identity
 from ccf.clients import CCFClient
 
+from utils import parse_ledger_url
```

Comment:
this isn't used.
also... too much cut and paste. make sure you update the copyright dates and any comments in the file.

Reply:
thank you, FYKI, parse_ledger_url is used in this module (line 90). Yes, I have updated the copyright date.